python/TestHarness/schedulers/TesterData.py

```diff
@@ -59,6 +61,10 @@ def processCommand(self, command):
 
         return process
 
+    def getCommand(self):
```

Comment:
I don't really like this for two reasons. First, a command is really part of a tester object, not a the scheduler data object. Second, this getter only works after you've called processCommand, it'll return blank otherwise. 

Caching the command isn't bad, but if you do it that way, it should be in the Tester object. 

The best way would be to make the Tester.getCommand() object not produce any side effects.